filtering scripts/filtering.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+from mdframe.reader import run, Config
+from filtering_helper import dataframe_check
+
+def filter_files(nutrition_subgroup : str = None, food_type : str = None, description : str = None, weight : float = None, 
+              unit : str = None, ingredients : list = None, project_name : str = None, rgbd_file_names : list = None, 
+              nutrition_facts_sources : list = None, texture_sources : list = None, quality : int = None, quality_comments : str = None,
+              merged : bool = None, textured : bool = None, started : str = None, ended : str = None) -> list[str]:
+    
+    """Filters the dataframes based on the given parameters
```

Comment:
It's great that you're writing docstrings for your functions. Very good practice!